hack/common.sh

```diff
@@ -4,6 +4,8 @@ if [ -f cluster-up/hack/common.sh ]; then
     source cluster-up/hack/common.sh
 fi
 
+export GOFLAGS=-mod=vendor
```

Comment:
Can you just move out all these extra mentions of `GOPROXY=off GOFLAGS=-mod-vendor` to this place and temove them there?

Reply:
Exporting `GOPROXY=off` will also disable it for bazel... is it intended?